compiler/src/codegen/compcore.re

```diff
@@ -3278,7 +3283,6 @@ let compile_imports = (wasm_mod, env, {imports}) => {
     );
 
   List.iter(compile_import, imports);
-  Import.add_memory_import(wasm_mod, "mem", grain_env_mod, "mem", false);
```

Comment:
@ospencer Removing this is causing your linking test for the import_memory flag. I have no idea why.